frontend/src/pages/index.js

```diff
@@ -40,6 +42,30 @@ const ProjectCard = ({ name, full_name, description, owner }) => {
   )
 }
 
+export const getStaticProps = async () => {
```

Comment:
I know there's a function in the gitea client doing the same thing though I preferred to make the commit in a single file